.github/CODEOWNERS

```diff
@@ -0,0 +1,12 @@
+/ @Jigsaw-Code/outline-owners
+
+/docs/       @Jigsaw-Code/outline-strings-owners
+/resources/  @Jigsaw-Code/outline-strings-owners
+*.json       @Jigsaw-Code/outline-strings-owners
+*.md         @Jigsaw-Code/outline-strings-owners
+*.png        @Jigsaw-Code/outline-strings-owners
+*.svg        @Jigsaw-Code/outline-strings-owners
+
+/cordova-plugin-outline/  @Jigsaw-Code/outline-networking-owners
+/third_party/             @Jigsaw-Code/outline-networking-owners
+/tools/                   @Jigsaw-Code/outline-networking-owners
```

Comment:
Could you add me or a group for `/src/www/model/`?

Reply:
Right now you're the sole member of `outline-owners`, so everything other than the exclusions below would have to go through you.  If you're trying to future-proof this in some way we could add a new group for `/src/www/model/`, but I wouldn't know what to call it since it's identical to `outline-owners`.

Reply:
outline-owners sounds good

Reply:
```suggestion
/tools/                   @Jigsaw-Code/outline-networking-owners
/src/www/model/           @Jigsaw-Code/outline-owners
```